### PR TITLE
Shifted the mail sending part to SendGrid

### PR DESCRIPTION
**Pull request by dibyadas**
The mail sending part is now handled by SendGrid instead of Flask-Mail. And made use of environment variables.

### CODE REVIEW

**Comment by DefCon-007** [comment]
@dibyadas Squash the commits

File: app.py [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+                      "Message:- " + form_msg + "\n")
+    # print("content")
+    mail = Mail(from_email=from_email, subject=subject,
+                to_email=to_email, content=content)
+    # print("mail init")
+    try:
+        response = sg.client.mail.send.post(request_body=mail.get())
+        # print("sent")
+    except urllib.HTTPError:
+        print("not sent")
```
Comment (DefCon-007) [inline app.py:32]: Add another except to catch all other exceptions and notify user when the mail is not sent

**Comment by dibyadas** [comment]
@DefCon-007 Should I squash-merge it in, since I have write access to the repository?

**Comment by DefCon-007** [comment]
For now squash the commits and force push and fix the other thing I mentioned.

**Pull request merged**